src/internal/converters/ics/ics.go

```diff
@@ -33,6 +34,11 @@ import (
 // originalStartTimeZone, reminderMinutesBeforeStart, responseRequested,
 // responseStatus, sensitivity
 
+const (
+	iCalDateTimeFormat = "20060102T150405Z"
+	iCalDateFormat     = "20060102"
+)
+
```

Comment:
can you move this to the `dttm` package, please?

Reply:
do we really need it in that package? If it's not used outside of this package it seems like just adding a level of indirection to look things up and adding more to that namespace

Reply:
Yeah, these are specific to ics export. They will not be used outside of that.